org.eclipse.xtext/src/org/eclipse/xtext/findReferences/TargetURIConverter.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
```

Comment:
can you use the original copyright header

Reply:
Didn't notice the template was different, will do